base/reduce.jl

```diff
@@ -952,32 +955,35 @@ julia> count(i->(4<=i<=6), [2,3,4,5,6])
 
 julia> count([true, false, true, true])
 3
+
+julia> count(>(3), 1:7, init=0x00)
```

Comment:
Maybe `init=0x03` or something to show how the behavior differs besides changing the return type.